propka/input.py

```diff
@@ -35,25 +35,36 @@ def open_file_for_reading(input_file):
     return file_
 
 
-def read_molecule_file(input_file, mol_container):
+def read_molecule_file(input_file, mol_container, filename=None):
```

Comment:
Using `filename` as an optional kwarg is simple and clean.

I would only add an example (e.g. with StringIO) to show why one might want to use it.

Reply:
I've added some docs examples in f672c63, I didn't go into detail about _why_ you'd want to use file-like object. If needed I can expand in that direction (although it might be a bit much for just one function's worth).